pkg/virt-api/webhooks/utils.go

```diff
@@ -78,6 +78,8 @@ type Informers struct {
 	NamespaceLimitsInformer cache.SharedIndexInformer
 	VMRestoreInformer       cache.SharedIndexInformer
 	DataSourceInformer      cache.SharedIndexInformer
+	VMIInformer             cache.SharedIndexInformer
```

Comment:
informers must not be added to virt-api. DataSource, VMRestore look suspicious too, NamespaceLimits may be ok to keep. This is wrong pattern which started at some point. Our webhooks should not use informers. One is listed in https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#common-architecture-flaws-to-avoid (this is much more expensive than just getting the objecst in the case of a webhook).